src/main/java/org/portfolio/ourverse/common/security/JwtAuthenticationFilter.java

```diff
@@ -0,0 +1,50 @@
+package org.portfolio.ourverse.common.security;
+
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+import org.springframework.util.ObjectUtils;
+import org.springframework.util.StringUtils;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class JwtAuthenticationFilter extends OncePerRequestFilter {
+
+    private static final String TOKEN_HEADER = "Authorization";
+    private static final String TOKEN_PREFIX = "Bearer ";
+    private final JwtTokenProvider jwtTokenProvider;
+
+    /*
+        헤더를 읽어 토큰을 통해 인증정보 생성.
+     */
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain) throws ServletException, IOException {
+        String token = resolveTokenFromRequest(request);
+
+        if(StringUtils.hasText(token) && jwtTokenProvider.validateToken(token)){
```

Comment:
```suggestion
        if (StringUtils.hasText(token) && jwtTokenProvider.validateToken(token)) {
```
if 문에 대한 java coding convention 은 위와 같습니다. java 개발자들끼리 style 을 정의한 coding convention 에 맞추어서 작성해보는 것은 어떨까요?
https://naver.github.io/hackday-conventions-java/#space-between-keyword-parentheses